### PR TITLE
Add torsion benchmark supplement

### PR DESCRIPTION
**Pull request by ntBre**
## New Submission Checklist

- [x] Created a new folder in the submissions directory containing the dataset
- [x] Added `README.md` describing the dataset see [here](https://github.com/openforcefield/qca-dataset-submission/tree/master/submissions/2020-03-26-OpenFF-Gen-2-Torsion-Set-6-supplemental-2) for examples
- [x] All files used to produce the dataset are included with a description
- [x] Dataset follows the QCSubmit schema defined for [Datasets](https://github.com/openforcefield/qcsubmit/blob/56680a7d3298b5d8962edcb840b0fdb34558c053/qcsubmit/datasets.py#L340), [OptimizationDatasets](https://github.com/openforcefield/qcsubmit/blob/56680a7d3298b5d8962edcb840b0fdb34558c053/qcsubmit/datasets.py#L1062) and [TorsionDriveDatasets](https://github.com/openforcefield/qcsubmit/blob/56680a7d3298b5d8962edcb840b0fdb34558c053/qcsubmit/datasets.py#L1225)
- [x] Dataset filename matches pattern `dataset*.json`; may feature a compression extension, such as `.bz2`
- [x] A PDF depicting the molecules is attached, in the case of torsiondrives this should include the highlighting of the central bond, this can be done automatically using [qcsubmit](https://github.com/openforcefield/qcsubmit/blob/56680a7d3298b5d8962edcb840b0fdb34558c053/qcsubmit/datasets.py#L854). 
- [x] QCSubmit validation passed
- [x] Made a new dataset entry in the mapping table in repository `README.md`
- [ ] Ready to submit!

### CODE REVIEW

File: submissions/2024-04-18-OpenFF-Torsion-Benchmark-Supplement-v1.0/README.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+  * SCF properties:
+	* dipole
+	* quadrupole
+	* wiberg_lowdin_indices
+	* mayer_indices
+
+<!-- References -->
+[tm]: https://openforcefield.atlassian.net/wiki/spaces/FF/pages/2603909164/Torsion+multiplicity
+[chembl]: https://www.ebi.ac.uk/chembl/
+[cura]: https://github.com/ntBre/cura
```
Comment (lilyminium) [inline submissions/2024-04-18-OpenFF-Torsion-Benchmark-Supplement-v1.0/README.md:68]: I understand that `cura` is still in active development and changes are being made at a reasonable rate. Could you please link to the version that you used, or the commit (e.g. https://github.com/ntBre/cura/tree/8deb89fcbc84e3688968deadfdf2ff682bfd1e19)? This makes it much easier to track in the future if someone wants to reproduce the work.

File: submissions/2024-04-18-OpenFF-Torsion-Benchmark-Supplement-v1.0/README.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+the [torsion multiplicity][tm] project. The molecules in this data set were The
+molecules in this data set were selected from the [ChEMBL 33][chembl] database
+with additional processing steps described below.
+
+## General Information
+
+* Date: 2024-04-17
+* Class: OpenFF Optimization Dataset
+* Purpose: Improve proper torsion training data in Sage
+* Name: OpenFF Torsion Drive Supplement v1.0
```
Comment (lilyminium) [inline submissions/2024-04-18-OpenFF-Torsion-Benchmark-Supplement-v1.0/README.md:16]: Could you please make sure this is the same as the heading and directory name?

File: submissions/2024-04-18-OpenFF-Torsion-Benchmark-Supplement-v1.0/README.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
+# OpenFF Torsion Benchmark Supplement v1.0
+
+## Description
+
+An optimization data set created to expand the benchmarking data for both
+existing Sage 2.2.0 proper torsion parameters and new parameters added through
+the [torsion multiplicity][tm] project. The molecules in this data set were The
```
Comment (lilyminium) [inline submissions/2024-04-18-OpenFF-Torsion-Benchmark-Supplement-v1.0/README.md:7]:
```suggestion
the [torsion multiplicity][tm] project. The
```

**Review by lilyminium — CHANGES_REQUESTED** [review]
Thanks Brent, mostly LGTM! I have two very nitpicky comments: one below, and another about naming. Could you please add the name "OptimizationDataset" (or "Optimization Dataset") somewhere in the name? Otherwise it sounds a lot like a TorsionDrive dataset, and quite similar to the TorsionDrive dataset you submitted yesterday.

**Comment by ntBre** [comment]
I think I got all of the requested changes, let me know if I missed anything!

**Comment by lilyminium** [comment]
Thank you! I think the `long_description_url` in the Dataset also needs updating now, but otherwise all looks good!

**Comment by ntBre** [comment]
Ah of course, thanks!

**Review by lilyminium — APPROVED** [review]
LGTM :-)

**Pull request merged**